src/Faker/Provider/gu_IN/Person.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Faker\Provider\en_US;
+
+class Person extends \Faker\Provider\Person
+{
+    protected static $maleNameFormats = array(
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}}',
+        '{{titleMale}} {{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}} {{suffix}}',
+        '{{titleMale}} {{firstNameMale}} {{lastName}} {{suffix}}',
+    );
+
+    protected static $femaleNameFormats = array(
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{titleFemale}} {{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}} {{suffix}}',
+        '{{titleFemale}} {{firstNameFemale}} {{lastName}} {{suffix}}',
+    );
+
+    protected static $firstNameMale = array(
+        'કુલદીપ','વિશાલ','ધર્મેશ','નીરજ','નીરવ','રાકેશ','રાજવીર','વૈભવ','હિતેન','પરેશ','અક્ષય','રાજુ','મંથન','પ્રશાંત','પ્રતિક','સાગર','વિવેક','પાર્થ','અર્જુન','ક્રિષ્ના','રાજ','ગોપાલ','ઉમેશ','દીપક','રોહન','હર્ષ'
```

Comment:
That's a very limited dataset. Faker needs more randomness.

Besides, you should provide a reference (a url in a `@link` phpDoc) containing the list you copy here, so that future maintainers know where the data come from and how to update it.